flixel/addons/effects/FlxEffectFade.hx

```diff
@@ -0,0 +1,154 @@
+package flixel.addons.effects;
+
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.geom.Point;
+
+/**
+ * This gradually fills the bitmapData with a color.
+ * 
+ * @author adrianulima
+ */
+class FlxEffectFade implements IFlxEffect
```

Comment:
I'm not sure this is really all that useful, can't you achieve pretty much the same effect by tweening the `.color` property?